src/main/java/com/example/KitchenProject/CookingLevel.java

```diff
@@ -0,0 +1,8 @@
+package com.example.KitchenProject;
+
+public enum CookingLevel {
```

Comment:
What do you think about replacing the cooking level with cooking time? Sth like `FAST`, `MEDIUM`, `LONG`? I think that the cooking level doesn't introduce any real value for the recipe, but I can imagine a use case when you would like to include fast recipes on weekdays or long ones only on weekends.

Reply:
After noticing that `Recipe` has `time` prop, you can disregard the comment above. But I will still reconsider if `level` is something valuable. Can you give me one possible use case here?